src/emu.rs

```diff
@@ -1246,7 +1261,7 @@ impl Emu {
                                 Some(n) => n,
                                 None => "not mapped".to_string(),
                             };
-                            println!("mem trace read -> '{}' 0x{:x}: 0x{:x}  map:'{}'", operand, addr, v, name);
+                            println!("\tmem_trace: rip = {:x} read -> '{}' 0x{:x}: 0x{:x}  map:'{}'", self.regs.rip, operand, addr, v, name);
```

Comment:
@sha0coder i apologize for "taking over" the trace/diff format and getting rid of `pos`, trying to get it consistent. we will probably want to add `pos` back later